rest-client/swagger.json

```diff
@@ -7635,6 +7670,14 @@
             "$ref": "#/definitions/Upload"
           }
         },
+        "pageSize": {
+          "type": "integer",
+          "description": "maximum number of records in each returned page"
+        },
+        "offsetKey": {
+          "type": "string",
+          "description": "start key for next page"
+        },
```

Comment:
See comments above about these properties... should have default values and document maximum value.